packages/x-date-pickers/src/internals/components/PickersArrowSwitcher/PickersArrowSwitcher.tsx

```diff
@@ -108,7 +108,7 @@ export const PickersArrowSwitcher = React.forwardRef(function PickersArrowSwitch
     elementType: PreviousIconButton,
     externalSlotProps: slotProps?.previousIconButton ?? componentsProps?.previousIconButton,
     additionalProps: {
-      size: 'small',
+      size: 'medium',
```

Comment:
The size of the icon rendered was equivalent to a `medium`
Do we want smaller icons or do we want to keep the current UI ?

Reply:
The current UI is `small` (34px), the proposed `medium` produces `40px`. 🤷‍♂️ 🙈 
I'm not sure we should be making this change at this moment... 🤔

Reply:
On my computer, to keep the same icon size, I need to put `medium`

Reply:
Ah, sorry. I might have looked at a cached version or just didn't have straight eyes.
Visually it's the same. 👌 
It's just the fact that now the height "requested by header content" is even higher (40px) than previously (34px), even though we force the max-height to `30px`.
Shouldn't be a problem now, but might cause certain issues down the line.
**Current:**
![Screenshot 2023-03-14 at 11 12 30](https://user-images.githubusercontent.com/4941090/224953023-fe4e3900-33b3-4625-9135-cea63349d280.png)
**This implementation:**
![Screenshot 2023-03-14 at 11 12 47](https://user-images.githubusercontent.com/4941090/224953097-295ed4a5-0bd2-4c3b-b997-420c13a46630.png)

Reply:
Indeed
Maybe @gerdadesign would have an opinion here between using a "medium" `IconButton` or a "small" `IconButton` for those icons by default.